package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "librato-client",
-  "version": "0.0.8",
+  "version": "0.0.9",
```

Comment:
Should this be a `major` version change since it introduces an exact version of `lodash`?

Reply:
Works for me